packages/core/src/RenderingEngine/WSIViewport.ts

```diff
@@ -428,6 +453,48 @@ class WSIViewport extends Viewport {
     this.refreshRenderValues();
   };
 
+  /**
+   * Converts a slide coordinate to a image coordinate using WSI utils functions
+   * @param point
+   * @returns
+   */
+  public worldToIndexWSI(point: Point3): Point2 {
+    const affine = this.viewer[Symbol.for('affine')];
```

Comment:
Can you do this as a constant?

Reply:
I mean the affine symbol.  Also, could you create a bug report for dicom-microscopy-viewer to make affine not use the symbol setup?  That would be worthwhile generally to improve interaction with different front ends.